### DISCUSSION TITLE
Is postgres-operator free to use for commercial purposes?

### DISCUSSION THREAD

**Discussion by mustafa-korkmaz** [zalando/postgres-operator#1596] [Q&A] [answered]
Sorry this might be a stupid question but we were considering to use[ Crunchy Data](https://github.com/CrunchyData/postgres-operator) and turned out it is meant to be used for free only for development purposes.

My question is, are we able to use this operator for free in our commercial projects?

**Comment by chrisdrew1** [comment]
The PGO Postgres Operator project source code is available subject to the Apache 2.0 license with the PGO logo and branding assets covered by our trademark guidelines.

**Reply by mustafa-korkmaz** [reply]
Thanks for the quick response, our aim is to use it within a private K8s cluster to automate Postgres installation & maintenance without exposing or providing any features of the operator to the third parties or our customers.

Within this context, can I accept your answer as a Yes? Explicitly asking this because Crunchy Data also has Apache 2.0 license but somehow it is not allowed for commercial usage.

**Comment by chrisdrew1** [comment, marked as answer]
I'm pretty sure both operators are under the apache 2.0 and cover your use case.  That's how I read both of the licenses, but your mileage may vary.

**Comment by Jasper-Ben** [comment]
Out of curiosity, would you mind explaining why you think that Crunchy Data's operator cannot be used commercially? It is licensed under the terms of the Apache-2.0 license which explicitly allows the commercial use. Whoever is telling you otherwise is either lying or has no clue what they are talking about.

Edit: Ok, they have a subscription based model for ootb features which are not licensed under Apache2.0. That's not uncommon. The base featureset however is ofc free to use for commercial purposes.
